www/templates/diary/list.html

```diff
@@ -27,45 +25,51 @@
         If you want to change this from collection-repeat to ng-repeat, you
         MUST test on android first to make sure that the scrolling works.
         -->
-		<div collection-repeat="trip in data.currDayTrips">
+		<div collection-repeat="tripgj in data.currDayTripWrappers">
 			<div class="list card" style="margin-top: 20px; margin-bottom: 20px;">
-			 <ion-item
-             style="padding: 10px;">
+			 <ion-item style="padding: 10px;" class="list-item" href="#/root/main/diary/{{tripgj.data.id}}">
+                {{ if}}
```

Comment:
what does this do? Doesn't it print the value of $scope.if? Is that what you want?